### ISSUE TITLE
Package installation fails: FileNotFoundError

### ISSUE THREAD

**Issue by dweindl** [copasi/python-petab-importer#2]
The recommended `pip install git+https://github.com/copasi/python-petab-importer.git` fails on Ubuntu 22.04 / Python 3.10.4 / setuptools-63.2.0 with:
```
Collecting git+https://github.com/copasi/python-petab-importer.git
  Cloning https://github.com/copasi/python-petab-importer.git to /tmp/pip-req-build-69ob06l8
  Running command git clone --filter=blob:none --quiet https://github.com/copasi/python-petab-importer.git /tmp/pip-req-build-69ob06l8
  Resolved https://github.com/copasi/python-petab-importer.git to commit 3bc83b3f652c56682dd2db12b754621867e62123
  Running command git submodule update --init --recursive -q
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [12 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-req-build-69ob06l8/setup.py", line 24, in <module>
          version=versioneer.get_version(),
        File "/tmp/pip-req-build-69ob06l8/versioneer.py", line 1678, in get_version
          return get_versions()["version"]
        File "/tmp/pip-req-build-69ob06l8/versioneer.py", line 1610, in get_versions
          cfg = get_config_from_root(root)
        File "/tmp/pip-req-build-69ob06l8/versioneer.py", line 338, in get_config_from_root
          with open(setup_cfg, "r") as cfg_file:
      FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-req-build-69ob06l8/setup.cfg'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by fbergmann** [comment]
I swear this was fine in 2021 when i wrote this. I'll have a look . in the meantime could you try installing the pypi package?

**Comment by dweindl** [comment]
> I swear this was fine in 2021 when i wrote this. I'll have a look . 

Yes, I'm rather sure it worked at some point... 

> in the meantime could you try installing the pypi package?

Thanks, I didn't recall it was available on pypi. `pip install copasi-petab-importer` works.

**Comment by MerktSimon** [comment]
pypi install also worked for me. However had to `pip install wheel` before.

**Comment by fbergmann** [comment]
it took a while, but this is now fixes with release 1.0.3
